src/backend/egl/display.rs

```diff
@@ -162,7 +168,10 @@ impl EGLDisplay {
             let mut minor: MaybeUninit<ffi::egl::types::EGLint> = MaybeUninit::uninit();
 
             wrap_egl_call(|| unsafe {
-                ffi::egl::Initialize(display, major.as_mut_ptr(), minor.as_mut_ptr())
+                println!(
+                    "Initialized EGL with result {:?}",
```

Comment:
I'd print this message in the map_err branch. Sadly inspect_err is still unstable.

Reply:
We could have a todo comment to remind us about that?

Reply:
Yeah probably.